recipe/meta.yaml

```diff
@@ -30,24 +32,25 @@ test:
     - tests
   requires:
     - pytest >=6.2,<7
-    - pytest-asyncio >=0.15,<1
     - pytest-benchmark >=3.4,<4
-    - pytest-cov >=2.11,<3
-    - pytest-describe >=1,<2
-    - pytest-timeout >=1.4,<2
```

Comment:
These pytest packages are still required https://github.com/graphql-python/graphql-core/blob/689a70d052a58277d20cc6b6348b0eb31ad3a113/pyproject.toml#L53-L57
Or is there a good point in not using them (with the simplified pytest command)?

Reply:
pytest-cov is not needed at all because we don't care about coverage reports.

For some reason, the only tests that are executed when running pytest are the benchmark tests, so the other pytest packages are not needed. We could either keep them for completeness or we leave a comment about why they are not needed.